one/commands/auth.py

```diff
@@ -102,9 +102,22 @@ def aws():
 
 
 @auth.command(help='Authentication using AWS IAM user.')
-def iam():
+@click.option('-i', '--access_key_id', default=None, envvar='AWS_ACCESS_KEY_ID', type=str, help='AWS access-key-id.')
+@click.option('-k', '--secret_access_key', default=None, envvar='AWS_SECRET_ACCESS_KEY', type=str, help='AWS secret-access-key.')
+@click.option('-r', '--region', default='us-east-1', envvar='REGION', type=str, help='AWS default region.')
```

Comment:
would be better to use `AWS_DEFAULT_REGION` instead of REGION, as it's the official variable for AWS cli.

Reply:
@adenot Should i also move `access_key_id` to `aws_access_key_id` and `secret_access_key` to `aws_secret_access_key`?

Reply:
no, the variables are fine as it's only for AWS anyway, but I meant change to envvar='AWS_DEFAULT_REGION'